site/quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quick Start Guide
+
+Use this Quick Start Guide to create a WebLogic deployment in a Kubernetes cluster.
+(Maybe add what users would do with this set up.)  
+
+(Add link to page that describes a very specific deployment following our [samples flow](http://aseng-wiki.us.oracle.com/asengwiki/display/ASDevWLS/2.0+Samples+Flow) with a Traefik load balancer and just one domain. RM, I'm not sure that this is needed in the Quick Start Guide. **This is a tutorial. What we need are the actual commands for each step listed.**)
+
+## 1.	Get the images and put them into your local registry.
+Get all the necessary images: Operator, Traefik. (Need a complete list of the required images and pointers to their locations, including the commands for retrieving the images and where users should locate them on their systems.)
+
+## 2. Create a Traefik (Ingress-based) load balancer.
+Use Helm to install a [Traefik](samples/charts/traefik/README.md) load balancer. (We can't just use a pointer to a script, we need to copy into this guide, the commands for each step.)
+
+## 3. Configure Kibana and Elasticsearch.
+Invoke the script to create these. (Again, list the commands here using an example.)
+
+## 4. Install the operator.
+* Create a namespace and service account for the operator (point to sample script that does this.)
+* Invoke the script to generate the credentials for the operator and add it to the the operator YAML file (all the values can be left as default).
+* Create the operator using `helm install`, and passing in the namespace, service account, and location of Elasticsearch (Point to Helm.)
+
+## 5. Prepare your environment for a domain.
+* Create a domain namespace
+* Create the Kubernetes secrets for the Administration Server boot credentials
+* Create a PV & PVC domain; use script
+* Configure the operator to manage the domains in the domain namespace using `helm upgrade`
+* Configure the Traefik load balancer to manage the domains in the domain namespace.
+  * Use `helm upgrade` to configure the load balancer to monitor the Ingresses in the domain namespace
+* Create the domain home
+* Create the Docker image for the domain home in the image or use the WebLogic binary image.
+  * Run WLST to create the domain in PV (remember to apply patch).
+
+## 6. Create a domain.
+* Edit the domain YAML file (can the defaults be used?)
+* Create the domain home for the domain
+  * For a domain home on PV – as in sample
+  * For a domain home in image, use the sample in the Docker GitHub project
```

Comment:
Must create the WebLogic domain home image and put it in the local repository

Reply:
OK.